Add breaking change for plugin signature (#775)
nushell/nushell.github.io: blog/2023-02-21-nushell_0_76.md

- `load-env` can not accidentally set `$env.PWD` anymore ([#7963](https://github.com/nushell/nushell/pull/7963))
- The `--numbered` flag to enumerate entries has been removed from `each`, `par-each`, `each while`, and `reduce`. Use the `enumerate` command added in 0.75 instead. ([#7777](https://github.com/nushell/nushell/pull/7777))

> - Plugin: `nu_plugin::Plugin::signature` is changed from `fn signature(&self) -> Vec<Signature>` to `fn signature(&self) -> Vec<PluginSignature>` to support examples on plugin commands.  Plugin authors have to update the relative structure to apply these changes, normally just need to make a global search/replace from `Signature` to `PluginSignature`, then rebuild and run with latest nushell version. ([#7984](https://github.com/nushell/nushell/pull/7984))

Reviewer: ```suggestion
- Plugin system: `nu_plugin::Plugin::signature` has changed from `fn signature(&self) -> Vec<Signature>` to `fn signature(&self) -> Vec<PluginSignature>` to support examples on plugin commands.  Plugin authors have to update the relative structure to apply these changes. Normally you just need to perform a global search/replace from `Signature` to `PluginSignature`, then rebuild and run with the latest nushell version. ([#7984](https://github.com/nushell/nushell/pull/7984))
```
